todo_app/app.py

```diff
@@ -1,24 +1,38 @@
-from todo_app.data.session_items import add_item, get_items
+import requests
+from todo_app.flask_config import Config
 from flask import Flask
 from flask import request
 from flask import render_template
 from werkzeug.utils import redirect
-from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
+trelloKey = app.config['TRELLO_KEY']
+trelloToken = app.config['TRELLO_TOKEN']
+toDoListId = app.config['TRELLO_TO_DO_LIST_ID']
+doneListId = app.config['TRELLO_DONE_LIST_ID']
 
-@app.route('/')
+@app.route('/', methods=['GET', 'POST'])
 def index():
-    return render_template('index.html', headers=['id', 'status', 'title'], results=get_items())
+    if request.method == 'POST':
+        toDoCardId = request.form['cardid']
+        url = f"https://api.trello.com/1/cards/{toDoCardId}/?key={trelloKey}&token={trelloToken}&idList={doneListId}"
+        response = requests.request("PUT", url)
+        return redirect('/')
+    if request.method == 'GET':
+        headers = {"Accept": "application/json"}
+        url = f"https://api.trello.com/1/lists/{toDoListId}/cards?key={trelloKey}&token={trelloToken}"
+        response = requests.request("GET", url, headers=headers)
+        return render_template('index.html', todos=response.json())
```

Comment:
Rather than just passing the json into the template directly, what we want to do is:
- define a python class to represent an item
- for each element in the returned json, create an instance of the item class
- pass those into the template.

Reply:
I've added a new class to get the cards